src/components/CheckEditor/CheckProbes.tsx

```diff
@@ -81,16 +82,20 @@ export default class CheckProbes extends PureComponent<CheckProbesProps, CheckPr
           invalid={!Validation.validateProbes(probes)}
           closeMenuOnSelect={false}
         />
-        <Container margin="xs">
-          <HorizontalGroup spacing="md">
+        <div
+          className={css`
+            margin-top: 0.5rem;
```

Comment:
I'm not sure about this being specified in `rem`, considering that we're using a 8px grid. But maybe I'm lacking the context for why it's necessary.

Reply:
Just muscle memory I need to forget. `rem` tends to result in a more stable layout when users vary their font size, zoom, etc, but consistency is more important

Reply:
I'm aware, [fructosedb](https://github.com/s-h-a-d-o-w/fructosedb) mostly uses `rem`. :)
Although I believe I checked out whether that's still true today and think that browsers have become very good at scaling pixel-based layouts appropriately. (I could be wrong about this though - if I was to start a fresh project right now, I'd double-check.)